components/runtime.rst

```diff
@@ -329,6 +329,9 @@ You can also configure ``extra.runtime`` in ``composer.json``:
         }
     }
 
+Then, update your Composer files (running ``composer update``, for instance),
```

Comment:
composer dump-autoload would be enough (or composer install btw)

Reply:
`composer update` can lead to unwanted changes :/

Reply:
Updated